draft-ietf-quic-transport.md

```diff
@@ -1198,12 +1198,23 @@ The fields of a GOAWAY frame are as follows:
 
 # Packetization and Reliability {#packetization}
 
-The maximum packet size for QUIC is the maximum size of the encrypted payload of
-the resulting UDP datagram.  All QUIC packets SHOULD be sized to fit within the
-path's MTU to avoid IP fragmentation.  The recommended default maximum packet
-size is 1350 bytes for IPv6 and 1370 bytes for IPv4.  To optimize better,
-endpoints MAY use PLPMTUD {{!RFC4821}} for detecting the path's MTU and setting
-the maximum packet size appropriately.
+The maximum packet size for QUIC is the maximum size of the entire UDP payload,
+including the public header, encrypted payload, and any authentication fields.
+All QUIC packets SHOULD be sized to fit within the path's MTU to avoid IP
+fragmentation. To optimize bandwidth efficiency, endpoints MAY use Path MTU
+Discovery ({{!RFC1191}}, {{!RFC1981}}) or Packetization Layer Path MTU
+Discovery ({{!RFC4821}}) for detecting the path's MTU, setting the maximum
+packet size appropriately, and storing the result of previous PMTU
+determinations. In the absence of these mechanisms, the recommended default
+maximum packet size is 1350 bytes for IPv6 and 1370 bytes for IPv4. 
```

Comment:
I don't remember any discussion concluding that those numbers are OK.  Are we assuming 1398 bytes is available to IP?

I find it a little strange that this is accounting for the IP and UDP headers.  I usually see MTU discussed in terms of the entire IP packet.  I would phrase this differently: "Prior to attempting PMTUD, or if PMTUD is not attempted, an endpoint SHOULD assume a path MTU of 1398 octets.  Assuming that there are no IPv6 extension headers or IPv4 options in use, this results in a UDP payload length of 1350 octets for IPv6 and 1370 octets for IPv4."

Or is the intent to actually say that either extension headers are forbidden or that elements that intend to add extension headers have to be able to pass larger packets?